core/include/thread.h

```diff
@@ -442,13 +453,39 @@ int thread_wakeup(kernel_pid_t pid);
  *
  * @return          obviously you are not a golfer.
```

Comment:
I never got that joke.

Reply:
Me neither. Maybe this is also not the best place for joking.

Reply:
https://www.youtube.com/watch?v=WDy9S8dxsE8

Reply:
I'd be open to keeping the joke in the compat wrapper for compatibility reasons.

Reply:
lebowski quote: https://www.youtube.com/watch?v=0CWfPUrmeiU

Reply:
I still don't get the relation of the function to Big Lebowski...

Reply:
I kept the joke for now. Maybe this is some discussion to have while drinking beer and outside of this PR. I feel drinking beer might be the key to understanding this.

Reply:
>  Maybe this is also not the best place for joking.

not this discussion again. :)

>  still don't get the relation of the function to Big Lebowski...

that's because you obviously don't golf.

Reply:
I guess the joke could also be supplanted with "I leave it to you Captain Obvious" ;-)